README.md

```diff
@@ -1,81 +1,76 @@
-### Debugger.html
+# Debugger.html
 
-This is a prototype debugger written without any XUL and based on React and Redux.
+This is a prototype debugger based on React and Redux.
 
 [![Build Status](https://travis-ci.org/jlongster/debugger.html.svg?branch=master)](https://travis-ci.org/jlongster/debugger.html)
 
-#### Getting Started
+## Getting Started
 
 * `npm install` - Install Dependencies
 * `npm run firefox` - Start Firefox
 * `npm start` - Start Debugger
 
 ![screen shot 2016-05-16 at 1 24 29 pm](https://cloud.githubusercontent.com/assets/254562/15297643/34575ca6-1b69-11e6-9703-8ba0a029d4f9.png)
 
-#### Getting Involved
+If you would like to open a specific Firefox instance or use a specific profile, first make sure it [has the appropate flags](#) enabled. Then run the following command and replace <Firefox.app> with your Firefox. For example, nightly would be `FirefoxNightly.app`.
```

Comment:
I'd prefer keeping this content in Miscellaneous as it's something that most open source contributors won't need.